src/renderer/components/swap/Swap.tsx

```diff
@@ -280,9 +280,18 @@ export const Swap = ({
   const chainFees$ = useMemo(() => fees$, [fees$])
   const approveFees$ = useMemo(() => approveFee$, [approveFee$])
 
+  const chainFeesORef = useRef<O.Option<SwapFees>>(O.none)
```

Comment:
[nitpick] `chainFeesORef` -> `prevChainFees` (just to have similar naming as we have at other places)